extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -67,6 +68,13 @@ class AmpIframe extends AMP.BaseElement {
         minTop);
   }
 
+  assertPlaceholder() {
```

Comment:
This should not be the case. Placeholder is styled automatically as `fill` - if not that should be fixed. But layout cannot be required here - it doesn't even have to be an AMP element.